### ISSUE TITLE
Add error information to info page

### ISSUE THREAD

**Issue by dkristia** [Paivola-Student-Innovation-Lab/DailyCrypt#37]
Add information on why file storing can fail, for example "This may be caused by the OPFS data being deleted by another tab. Reload the page to fix this."
